File: Examples/CSharp/Programming-Documents/Joining-Appending/JoinContinuous.cs
using System;
using System.IO;

using Aspose.Words;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Joining_and_Appending
{
    class JoinContinuous
    {
        public static void Run()
        {
            //ExStart:JoinContinuous
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_JoiningAndAppending();
            string fileName = "TestFile.Destination.doc";

            Document dstDoc = new Document(dataDir + fileName);
            Document srcDoc = new Document(dataDir + "TestFile.Source.doc");

            // Make the document appear straight after the destination documents content.
            srcDoc.FirstSection.PageSetup.SectionStart = SectionStart.Continuous;

            // Append the source document using the original styles found in the source document.
            dstDoc.AppendDocument(srcDoc, ImportFormatMode.KeepSourceFormatting);
            dataDir = dataDir + RunExamples.GetOutputFilePath(fileName);
            dstDoc.Save(dataDir);
            //ExEnd:JoinContinuous
            Console.WriteLine("\nDocument appended successfully with continous section start.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Joining-Appending/JoinContinuous.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Reflection
Imports System.Collections
Imports System.IO
Imports System.Text

Imports Aspose.Words.Lists
Imports Aspose.Words.Fields
Imports Aspose.Words

Public Class JoinContinuous
    Public Shared Sub Run()
        ' ExStart:JoinContinuous
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_JoiningAndAppending()
        Dim fileName As String = "TestFile.Destination.doc"

        Dim dstDoc As New Document(dataDir & fileName)
        Dim srcDoc As New Document(dataDir & "TestFile.Source.doc")

        ' Make the document appear straight after the destination documents content.
        srcDoc.FirstSection.PageSetup.SectionStart = SectionStart.Continuous

        dataDir = dataDir & RunExamples.GetOutputFilePath(fileName)
        ' Append the source document using the original styles found in the source document.
        dstDoc.AppendDocument(srcDoc, ImportFormatMode.KeepSourceFormatting)
        dstDoc.Save(dataDir)
        ' ExEnd:JoinContinuous
        Console.WriteLine(vbNewLine & "Document appended successfully with join continous option." & vbNewLine & "File saved at " + dataDir)
    End Sub
End Class
